src/Knp/Menu/Matcher/Voter/CallbackVoter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Knp\Menu\Matcher\Voter;
+
+use Knp\Menu\ItemInterface;
+
+/**
+ * Voter based on a callback
+ */
+class CallbackVoter implements VoterInterface
```

Comment:
please make this class final so that I don't have to maintain BC for child classes. There is no valid reason to inherit from it anyway.